### PR TITLE
add subcommand "replicaset" for "kubectl create"

### PR DESCRIPTION
**Pull request by dixudx**
**What this PR does / why we need it**:
xref [kubernetes/kubectl#12](https://github.com/kubernetes/kubectl/issues/12)

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
/assign @mengqiy @pwittrock

**Release note**:

```release-note
add subcommand "replicaset" for "kubectl create"
```

### CODE REVIEW

**Comment by cblecker** [comment]
/ok-to-test
Please note: We are currently in the 1.8 release code freeze and as such this PR will not be able to be merged until that lifts. Thanks for your contribution!

**Comment by dixudx** [comment]
@mengqiy @pwittrock Rebased. PTAL. Thanks.

File: pkg/kubectl/cmd/create_replicaset.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(CreateReplicaSet(f, cmdOut, cmd, args))
+		},
+	}
+	cmdutil.AddApplyAnnotationFlags(cmd)
+	cmdutil.AddValidateFlags(cmd)
+	cmdutil.AddPrinterFlags(cmd)
+	cmdutil.AddGeneratorFlags(cmd, cmdutil.ReplicaSetAppsV1beta2GeneratorName)
+	cmd.Flags().Int32("replicas", 1, "the number of desired replicas")
+	cmd.Flags().StringSlice("image", []string{}, "Image name to run.")
```
Comment (apelisse) [inline pkg/kubectl/cmd/create_replicaset.go]: As mentionned in the initial issue, this parameter should be factorized somewhere, as you keep recreating it.

**Comment by apelisse** [comment]
I'm not opposed to that code, but you have many failing tests :-)

/retest

**Pull request closed without merging**